Makefile

```diff
@@ -1,40 +1,50 @@
 dune = dune $(1) $(DUNE_$(1)_FLAGS)
 
-all:
+elpi/dune: elpi/dune.in
+	@rm -f $@
+	@echo "; generated by make, do not edit" > $@
+	@if $$(coqc --version | grep -q "8.19\|8.20") ; then \
+	  sed -e 's/@@STDLIB_THEORY@@//' $< >> $@ ; \
+	else \
+	  sed -e 's/@@STDLIB_THEORY@@/(theories Stdlib)/' $< >> $@ ; \
```

Comment:
@proux can't we provide a stdlib-shim package for coq < 9 ?
Unless elpi is the only package needing this

Reply:
Well, coq-elpi is the only thing in the whole Coq CI that need this, so not sure it would be worth it. In any case, we should get rid of that dependency here (or at least isolate it to apps that really need it and put them in a separate coq-elpi-extra-apps or so package).